server/TypeDBServer.java

```diff
@@ -210,66 +226,67 @@ protected void serve() {
             server.awaitTermination();
         } catch (InterruptedException e) {
             // server is terminated
-            close();
-            Thread.currentThread().interrupt();
+            try {
+                close();
+                System.exit(0);
```

Comment:
we expect every user of `close()` to have their own error handling and exit code use.